cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -404,30 +404,60 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             return parseInt(this.$('#due_in').val());
         },
 
+        showProjectedDate: function() {
+            var startDate = DateUtils.parseDateFromString(this.model.get('start'))
+            this.$("#startDate").text(gettext(startDate.toDateString()));
```

Comment:
This doesn't need the gettext call. It is just a date and wouldn't be translated. I believe the browser along with the date library will convert it to the format of the locale.